pydantic/main.py

```diff
@@ -220,7 +211,13 @@ def __instancecheck__(self, instance: Any) -> bool:
         return hasattr(instance, '__pydantic_validator__') and super().__instancecheck__(instance)
 
 
-class BaseModel(_repr.Representation, metaclass=ModelMetaclass):
+class _BaseModelBase(_repr.Representation):
```

Comment:
I assume this is here to be used by `RootModel`? Otherwise I'd rather not have it.